### ISSUE TITLE
User having issues with mods and vips

### ISSUE THREAD

**Issue by udoprog** [udoprog/OxidizeBot#164]
A user reported to me that mods and vips are not being updated - even when forced with[ `refresh-mods`](https://setbac.tv/help?q=refresh-mods).

I'm doing some maintenance work which should help troubleshoot this. But so far I've not been able to reproduce it.

**Comment by udoprog** [comment]
Got it now:

```
Unhandled notice msg_id=Some("unrecognized_cmd") Unrecognized command: /mods
```

**Comment by udoprog** [comment]
Fix had been pushed. There's still one more thing that needs to be done before it can be released. Currently there's a sporadic error when updating stream information I'd like to get rid of.

**Comment by udoprog** [comment]
This has been fixed, and has been merged into `master`. There's now also a pre-release available that you can try out:

https://github.com/udoprog/OxidizeBot/releases/tag/nightly
